win32/Lib/win32timezone.py

```diff
@@ -878,38 +921,43 @@ def GetTZCapabilities():
 
 
 class DLLHandleCache:
-    def __init__(self):
-        self.__cache = {}
+    def __init__(self) -> None:
+        self.__cache: dict[str, int] = {}
 
-    def __getitem__(self, filename):
+    def __getitem__(self, filename: str) -> int:
         key = filename.lower()
         return self.__cache.setdefault(key, win32api.LoadLibrary(key))
 
 
 DLLCache = DLLHandleCache()
 
 
-def resolveMUITimeZone(spec):
+def resolveMUITimeZone(spec: str) -> str | None:
     """Resolve a multilingual user interface resource for the time zone name
 
     spec should be of the format @path,-stringID[;comment]
     see http://msdn2.microsoft.com/en-us/library/ms725481.aspx for details
+
+    >>> import sys
+    >>> result = resolveMUITimeZone('@tzres.dll,-110')
+    >>> expectedResultType = [type(None),str][sys.getwindowsversion() >= (6,)]
+    >>> type(result) is expectedResultType
+    True
```

Comment:
I had accidentally removed a bit too much in https://github.com/mhammond/pywin32/pull/2085/files#diff-e5d2b4683af028d6a25d25c862c248214190080072e6f07171106c56dd7812dc